ui/ui_dev_server.js

```diff
@@ -8,7 +9,14 @@ const app = next({ dev })
 const handle = app.getRequestHandler()
 var httpProxy = require('http-proxy');
 
-var proxy = httpProxy.createProxyServer({});
+var proxy = httpProxy.createProxyServer({
+  target: {
+    host: "localhost",
```

Comment:
Avoid this hardcoding, if we can.

Reply:
The only scenario i can think of where it might change is when we do not run meshery server in `localhost:9081` and i don't think someone who wants to just contribute will do that 🤔 
What do you think ?

Reply:
`ui_dev_server` is only used for development purposes, and anyways the UI dev server has no way to know the address of the Meshery server, so no way around it.

Looks good Karthik.